tensorflow/inference/docker/2.3.0/py36/Dockerfile.eia

```diff
@@ -0,0 +1,125 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+# Specify LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+LABEL dlc_major_version="1"
+
+# Add arguments to achieve the version, python and url
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.6.13
```

Comment:
Please verify that this dockerfile should build for Python 3.6, and not Python 3.7.